client/tests/e2e/TestReview.e2e.test.js

```diff
@@ -70,6 +70,8 @@ describe('Test Review page', () => {
         url: '/data-management'
       },
       async page => {
+        page.setDefaultTimeout(60000);
+
```

Comment:
Also orthogonal to this PR but I found that this test was flakey with the timeout. This did not completely resolve though.